Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Specify the version of Go to use
+FROM golang:1.19
+
+ENV GO111MODULE=on \
+    CGO_ENABLED=0 \
+    GOOS=linux \
+    GOARCH=amd64
+
+WORKDIR /conformance
+COPY go.mod .
```

Comment:
If this Docker image is going to be published somewhere, suggest condensing all copy and run operations:

```dockerfile
COPY . .

RUN go mod download && go build -o /conformance/run
```

Then you can use a .dockerignore (same syntax as .gitignore) to exclude files that you don't want copied.

This is better because:

1. If you add more files in the future you don't need to remember to update the Dockerfile
2. Each operation creates a new layer in the published Docker container. THe more you can condense, the less layers.